vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -1518,4 +1522,108 @@ winrt::Microsoft::ReactNative::ComponentView WindowsTextInputComponentView::Crea
   return winrt::make<WindowsTextInputComponentView>(compContext, tag, reactContext);
 }
 
+void WindowsTextInputComponentView::autoCapitalizeOnUpdateProps(const std::string &capitalizationType) noexcept {
+  /*
+    Possible values are:
+     Characters - All characters.
+     Words - First letter of each word.
+     Sentences - First letter of each sentence.
+     None - Do not autocapitalize anything.
+  */
+
+  BSTR bstr = nullptr;
+  winrt::hresult hr = m_textServices->TxGetText(&bstr);
```

Comment:
```suggestion
  winrt::check_hresult(m_textServices->TxGetText(&bstr));
```

and then no `SUCCEEDED(hr)`